### ISSUE TITLE
Profile feeds

### ISSUE THREAD

**Issue by cryptoquick** [mikedilger/gossip#430]
When I go to someone's profile page, I can't see their notes. It'd be nice to see their notes, and to be able to send them DMs.

**Comment by mikedilger** [comment]
If you click their name you can pick an option to view their notes.

**Comment by cryptoquick** [comment]
Ah! Do you think it might make sense to have the feed from "View Their Posts" right below their profile info in their Account page, so it's not a separate view, but a combined view?

**Comment by mikedilger** [comment]
I'm open to UX changes here. I put a lot of thought into making functionality available and working right, but very little thought into the UX.  I think it is too "hidden" to click the user's name, and maybe @cryptoquick is right that the posts should just be on this page.

@dtonon is heading up UI/UX so I'll leave this to him.

**Comment by dtonon** [comment]
Yes, in the profile we absolutely need to "explode" the dropdown linked to the nym, to make some actions more prominent.

Putting the user's notes below the profile is a classic solution, we can go with this path if egui doesn't make it too hard. @bu5hm4nn what do you think? We need to take a decision about this, and then I will go on with the proposal design.
